pkg/diagnoser/diagnoser.go

```diff
@@ -6,15 +6,17 @@ import "github.com/Azure/aks-periscope/pkg/interfaces"
 type Type int
 
 const (
+	//ConfigValidator defines ConfigValidator Diagnoser Type
+	ConfigValidator Type = iota
 	// NetworkConfig defines NetworkConfig Diagnoser Type
-	NetworkConfig Type = iota
```

Comment:
💡 Wondering why `iota` is removed? I believe it is the `const` use purposes, do you know much in detail that removing this will not cause issues existing feature please?

Reply:
Sorry, this was unintentional. I will add it back.

Reply:
https://golang.org/ref/spec#Iota looks like as long as the first entry in the const has "= iota" defined then additional entries with "= iota" won't have any effect (within that same const). 

Don't think it really matters in this case, but general best practise is to add new values to the end of the enum so as not to change any of the existing mappings, if adding multiple at once then alphabetical ordering of the _newly added_ entries is probably a good idea (in lieu of any other ordering).